src/services/user.service.js

```diff
@@ -0,0 +1,33 @@
+const { User } = require('../models/User.model');
+
+const getAll = async () => {
+  return User.findAll();
+};
+
+const get = async (id) => {
+  return User.findByPk(id);
+};
+
+const create = async (user) => {
+  return User.create(user);
+};
+
+const update = async (user) => {
+  const { id } = user;
+
+  return User.update({ ...user }, { where: { id } });
```

Comment:
The `update` function should return the updated user object. Consider using `User.findByPk(id)` after the update to retrieve and return the updated user.